### ISSUE TITLE
Difficult to observe the loss value during training in the Terminal

### ISSUE THREAD

**Issue by phongnhhn92** [facebookresearch/Detectron#17]
Hi guys ! First of all, amazing work !!! 
I know this is not a bug but I just want to say that it is very difficult for me to observe how the loss value changes during training. I think representing the loss values in json format is pretty bad idea. I would like to alter this for better visualization (like Keras). I will update my progress in here.
![screenshot from 2018-01-24 13-14-01](https://user-images.githubusercontent.com/11288381/35314240-356681b8-0108-11e8-8c5c-4b9ccbe2b0bc.png)

**Comment by rbgirshick** [comment]
The idea is that you can easily parse the training output as json and then plot the metrics of interest. We are unlikely to change this.

**Comment by edumucelli** [comment]
On Keras you can easily get the JSON version of the [history](https://keras.io/callbacks/#history). It will be stored in a variable for later usage as a `dict`, which can be easily transformed into a JSON ("to plot the metrics of interest"). However during the training the printing does not need to be pure JSON, it could be in a more readable form.
